poetry.lock

```diff
@@ -2028,13 +2028,13 @@ openapi-schema-validator = ">=0.4.2,<0.5.0"
 
 [[package]]
 name = "packaging"
-version = "23.0"
+version = "23.2"
```

Comment:
Should we update it straight to the latest 24.2?

Reply:
Not sure. I was going for the safer route to keep things as compatible with both poetry v1 and v2 and other packages. We can try.

Reply:
Seems to work, too. Let's see what the tests say.